modules/plugins/utility/visual-multi/config.nix

```diff
@@ -0,0 +1,15 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}: let
+  inherit (lib.modules) mkIf;
+  cfg = config.vim.utility.visual-multi;
+in {
+  config = mkIf cfg.enable {
+    vim = {
+      startPlugins = [pkgs.vimPlugins.vim-visual-multi];
```

Comment:
Unless this plugin has weird build steps (e.g. some actual build step other than just copying plugin files), consider adding the plugin to flake inputs and building it from source to better control plugin source. This process is described in [the online manual](https://notashelf.github.io/nvf/index.xhtml#sec-additional-plugins)